Makefile.am

```diff
@@ -233,6 +233,10 @@ BATS = \
 	test/functional/update/slow-server/test.bats \
 	test/functional/update/statedir-bad-hash/test.bats \
 	test/functional/update_v2/update-client-certificate.bats \
+	test/functional/update_v2/update-delta-manifest-invalid.bats \
+	test/functional/update_v2/update-delta-manifest-no-iterative.bats \
+	test/functional/update_v2/update-delta-manifest-and-full.bats \
+	test/functional/update_v2/update-delta-manifest.bats \
```

Comment:
since there is already a PR that removes the update_v2 directory I'd suggest to add this directly to the update directory.

Reply:
Good idea, I will update this.